saleor/product/models.py

```diff
@@ -29,6 +29,8 @@ class Category(MPTTModel):
     parent = models.ForeignKey(
         'self', null=True, blank=True, related_name='children',
         on_delete=models.CASCADE)
+    background_image = VersatileImageField(
+        upload_to='category_backgrounds', blank=True, null=True)
```

Comment:
Shouldn't we rather use hyphen here? Similar problem was mentioned in #1553. Same below.